### DISCUSSION TITLE
百人課程_Python程式設計

### DISCUSSION THREAD

**Discussion by Lipunpun** [1101-GS4719-C/Python-Community-GS4719-C#1194] [General]
![image](https://user-images.githubusercontent.com/125544059/229041367-6d20f006-3e69-49b8-9a0a-1f23d6c97205.jpeg)

大家應該沒有忘記，學期初拿密碼卡的場景吧，剛開始真的沒有想到這堂課那麼熱門，不是隨便選隨便上嗎？（沒我不想被打😆）

上了半個學期下來也大概知道為什麼了，除了過去分數很甜以外，老師上課內容真的對程式新手很友善，從頭細講安裝Python、運用git，老師提供很多學習資源，老師也用聊天機器人和我們有所互動，上課真的蠻好玩的。

總之我認為那麼多人修，真的是老師懂教，雖然三個小時的課真的很難集中注意力，但這堂課讓我從看到命令格式字元，就以為電腦要中毒的程式白癡，變成不會怕命令提示字元的程式白癡（應該算是有進步？），希望未來課堂可以學到更多。

**Comment by Lipunpun** [comment]
我是土木系的學生，希望在未來可以學到更多有關Python的應用，說不定可以早日脫離苦海，感受科技業的溫暖！但其實土木沒有不好啦個人是覺得很有趣，有的工地其實蠻壯觀的。

**Comment by CCello** [comment]
想當初怕學分數不夠拚死拚活的去加選修一堆課，好在這堂python課有加選上!
老師講得也不錯一些底層的概念都有教到，期待課程後段關於python應用方面的分享和講解。

**Comment by pigeon9975** [comment]
我是大氣系的學生，我們系上雖然有python的課程，但似乎比起語法，我們的課更著重在套件應用和繪圖上面，像我這種連迴圈都不太會跑的人真的聽了頭很痛，還好這堂課有加選上，填補我對基本語法的不足！
希望未來可以越來越進步，將這堂課學到的知識和系上課做結合，變成python小達人(？

**Comment by orianna901001** [comment]
當初覺得初選能上這堂課的我真的是天選之人XD
老師收一百學生然後線上教學的方式，讓我覺得這種教學方法真的很讚，鼓勵那些真的想學python以及需要學分的人，看每堂老師上課這麼認真的樣子，相信或多或少也會更認真學習！

**Comment by ALAN30359704** [comment]
雖然我的期中考考的沒有很理想，但我認為在這門課有學到如何用python來表達我的想法，在程式語言裡，我覺得最困難的就是要如何把想法實際展現出來，希望期末可以考的更好！

**Comment by llh26** [comment]
當時超怕自己選不上的，因為之後專題可能會用到python，所以我真的是拼死拼活也要選上，還在開學第一週就去申請學分學程，只為了拿到神聖的密碼卡。目前已經上了半學期的課程，真的跟一開始比起來有小進步了！至少不是菜鳥，希望自己在上完這學期的課程後，可以自己去研究課程以外的領域，看懂別人的程式碼💪🏻

**Comment by RogelioKG** [comment]
讓我分享一下這個更壯觀的視角。
![79106](https://github.com/NCU-GS4719-Python/Python-Community/assets/86526364/4fe97f4b-07d4-4b9e-9ea8-7eaeb6d193f9)
小弟大一，所以還是頭一次看到選課能有如此光景。
想到當初大一上，為了上家凱老師的 AI 導論，也是特別跑來拿神聖的密碼卡。
沒有後悔填這門學程，能遇到這樣的老師真的是太好了😌
